src/20_Components/amp-form.html

```diff
@@ -0,0 +1,78 @@
+<!---{
+    "experiment": true,
+    "component": "amp-form",
+    "draft": true
+  }--->
+<!--
+  #### Introduction
+
+  The [amp-form](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md0) extension allows the usage of forms and input fields in an AMP document. `amp-form` allows HTTP and XHR (XMLHttpRequest) form submissions. A HTTP form submission loads a new page, while a XHR form submission doesn't require a page-reload.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    Import the library.
+  -->
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script> 
+  <!-- 
+    Customize success or error messages by using `amp-form-submit-success` and `amp-form-submit-error` classes.
+   -->
+  <style amp-custom>
```

Comment:
suggestions: 

* add more space to error message (we never use 4px as a margin, always either 8px or 16px).
* use color names to make it more obvious to readers 
* the large font for error messages looks weird

```
  <style amp-custom>
        form.amp-form-submit-success [submit-success],  
          form.amp-form-submit-error [submit-error]{
            margin-top: 16px;
        }
        form.amp-form-submit-success [submit-success] {
            color: green;
        }
        form.amp-form-submit-error [submit-error] {
            color: red;
        }
    </style>
```